ingestion/src/metadata/ingestion/source/dynamodb.py

```diff
@@ -109,15 +109,11 @@ def ingest_tables(self, next_tables_token=None) -> Iterable[OMetaDatabaseAndTabl
     def get_columns(self, column_data):
         for column in column_data:
             try:
-                if "S" in column["AttributeType"].lower():
+                if "S" in column["AttributeType"].upper():
                     column["AttributeType"] = column["AttributeType"].replace(" ", "")
                 parsed_string = ColumnTypeParser._parse_datatype_string(
                     column["AttributeType"].lower()
                 )
-                if isinstance(parsed_string, list):
-                    parsed_string = {}
-                    parsed_string["dataTypeDisplay"] = str(column["AttributeType"])
-                    parsed_string["dataType"] = "UNION"
```

Comment:
Why did we remove this code?

Reply:
We need this code in case if the data returns a UNION from dynamodb and it gets parsed as list instead of dictionary

Reply:
This code was reused from glue.py which returned list in some cases. For DynamoDB a list is never returned so this code is not relevant here